scripts/.storybook/main.js

```diff
@@ -59,10 +59,20 @@ module.exports = {
       ],
     });
 
+    config.module.rules.push({
+      test: /pikaday\/css\/pikaday.css$/,
+      loaders: ['style-loader', 'css-loader'],
+    });
+
     // CSS Modules
     config.module.rules.push({
       test: /\.css$/,
-      exclude: [/node_modules/, /\.global\.css$/, /forma-36-fcss/],
+      exclude: [
+        /node_modules/,
+        /pikaday\/css\/pikaday.css$/,
```

Comment:
This should be covered by `/node_modules/` no? 🤔

Reply:
I think maybe it's not being covered because the import doesn't mention `node_modules`? 

`import 'pikaday/css/pikaday.css';`